templates/tftmpl/golden_test.go

```diff
@@ -87,18 +87,33 @@ func TestNewFiles(t *testing.T) {
 				Task: task,
 			},
 		}, {
-			Name:   "terraform.tfvars.tmpl",
+			Name:   "terraform.tfvars.tmpl (services condition)",
 			Func:   newTFVarsTmpl,
 			Golden: "testdata/terraform.tfvars.tmpl",
 			Input: RootModuleInputData{
-				Providers: []hcltmpl.NamedBlock{hcltmpl.NewNamedBlock(
-					map[string]interface{}{
-						"testProvider": map[string]interface{}{
-							"alias": "tp",
-							"attr":  "value",
-							"count": 10,
-						},
-					})},
```

Comment:
provider configs weren't necessary for the test-case so I removed it just to trim down the test-case a bit